### DISCUSSION TITLE
Identity services

### DISCUSSION THREAD

**Discussion by andrei0x309** [gitcoinco/passport#137] [Q&A] [answered]
I noticed that the passport repo from a UX standpoint is very similar to the trust page on gitcoin website. 
One thing I wanted to ask if is there are any plans to add Idena too like it was on the trust page?
Also, IMO Google, FB Twitter, and ENS are poor ways of verifying identity.  I don't think any of those has a higher Sybil than the phone verification that was also axed I see. 
Would like to hear some thoughts on this, thanks.

**Comment by QuickMythril** [comment, marked as answer]
Idena has been added, so i guess this can be marked as "answered" or closed.

**Reply by Shalipoppi** [reply]
let's see how it goes

**Reply by Shalipoppi** [reply]
hnnn

**Comment by andrei0x309** [comment]
Yeah saw that I'll do this now.

**Reply by Shalipoppi** [reply]
same as well
